src/main/java/uk/gov/hmcts/reform/jobscheduler/services/JobsService.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.jobscheduler.services;
+
+import org.apache.commons.lang.NotImplementedException;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.jobscheduler.model.Job;
+
+@Service
+public class JobsService {
```

Comment:
It would be better to keep this as an interface and benefit from using polymorphism.

Reply:
no benefits from doing that

Reply:
Controller could expect a `JobService` where configured implementation would be injected by spring. This is what DI is about.

Reply:
We need only one implementation atm. Tests use mock

Reply:
That's right - there's no other implementation of this internal `JobsService`. Hiding it behind an interface would be premature and would most likely be an overhead.